README.md

```diff
@@ -101,6 +103,24 @@ Also, you can create TLV objects directly using the `bertlv.TLV` struct (less pr
 	}
 ```
 
+### Unmarshaling to structs
+
+The `bertlv.Unmarshal` function allows you to unmarshal TLV data directly into Go structs using struct tags. Fields can be mapped to TLV tags using the `bertlv` struct tag:
+
+```go
+type EMVData struct {
+    DedicatedFileName   []byte `bertlv:"84"`
+    ApplicationTemplate struct {
+        ApplicationID                string `bertlv:"4F"`       // Will be converted to HEX string
+        ApplicationLabel             string `bertlv:"50,ascii"` // Will be converted to ASCII string
+        ApplicationPriorityIndicator []byte `bertlv:"87"`
```

Comment:
Would these bytes be encoded? Hex?

Reply:
the source is always []byte, so if destination is []byte - then no decoding. Just assignment. If the destination is string, then by default we convert it into Hex. With `ascii` option in the field tag, we will convert it into ASCII (hopefully :).

Reply:
Makes sense.